### PR TITLE
Bg 21 one js file loads others

### PR DESCRIPTION
**Pull request by romelu**

### CODE REVIEW

File: code/js/rcalc-constant-html-class.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// classes
```
Comment (palomino7) [inline code/js/rcalc-constant-html-class.js]: For constants you should create new folder: constant.

File: code/js/rcalc-constant-number-utility.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const HEIGHT_DEFAULT_55_PX = 55;
+// size large in px
+const FONT_SIZE_LARGE_70_PX = 70;
+const FONT_SIZE_LARGE_30_PX = 30;
+const WIDTH_LARGE_100_PX = 100;
+const WIDTH_LARGE_305_PX = 305;
+const WIDTH_LARGE_410_PX = 410;
+const HEIGHT_LARGE_100_PX = 100;
+// zero﻿
+const NUMBER_ZERO = 0;
```
Comment (palomino7) [inline code/js/rcalc-constant-number-utility.js]: Maybe only NUMBER_ZERO should be in file number utility. Others are related with styling according to SIZING function. Try to do it better.

File: code/js/rcalc-constant-string-utility.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// adds to a value px, making it work to change atributes set in px
+const SUFFIX_PX = "px";
+// empty string
+const STRING_EMPTY = "";
+// colors
+const COLOR_BLACK = "#000000";
```
Comment (palomino7) [inline code/js/rcalc-constant-string-utility.js]: Again ... colors are styling related.

File: code/js/rcalc-loader.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+// loader injects js files in to the html file <head> tag
+function loadJsScript(jsFile) {
```
Comment (palomino7) [inline code/js/rcalc-loader.js]: It is path to js file. Not jsFile.

File: code/js/rcalc-constant-html-class.js (hunk 1 above)
Comment (romelu) [reply code/js/rcalc-constant-html-class.js]: folder has been created

File: code/js/rcalc-constant-number-utility.js (hunk 2 above)
Comment (romelu) [reply code/js/rcalc-constant-number-utility.js]: all constants related to styling in 1 js file called rcalc-constant-html-css

File: code/js/rcalc-constant-string-utility.js (hunk 3 above)
Comment (romelu) [reply code/js/rcalc-constant-string-utility.js]: all constants related to styling in 1 js file called rcalc-constant-html-css

File: code/js/rcalc-loader.js (hunk 4 above)
Comment (romelu) [reply code/js/rcalc-loader.js]: parameter in the function changed from jsFile to jsFilePath

**Pull request merged**